tldw_Server_API/app/core/Chat/README.md

```
@@ -127,6 +127,29 @@ FastAPI endpoint (app/api/v1/endpoints/chat.py)
 
 ---
 
+## Phase 2b Run-First Posture
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)